src/main/java/Duke.java

```diff
@@ -1,10 +1,163 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static int inputCount = 0;
+    public static Task[] tasks = new Task[100];
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String input;
+        Scanner in = new Scanner(System.in);
+
+
+        System.out.println("Hello! I'm Duke\n" + "What can I do for you?\n");
+
+        input = in.nextLine();
+
+        while (!(input.equals("bye"))) {
+
+            if (input.equals("list")) {
+                input = printList(in);
+
+            } else if (input.contains("done")) {
+                input = testTaskDone(input, in);
+
+            } else if (input.contains("todo")) {
```

Comment:
Storing "bye" / "list" / ... as constants above and naming them as
`private final String INPUT_BYE = "bye"` or the like could be more readable.